libc/src/__support/threads/linux/thread.cpp

```diff
@@ -517,4 +517,6 @@ void thread_exit(ThreadReturnValue retval, ThreadStyle style) {
   __builtin_unreachable();
 }
 
+pid_t Thread::get_uncached_tid() { return syscall_impl<pid_t>(SYS_gettid); }
```

Comment:
Can we clean up the number of places that we call `SYS_gettid` directly?  Looks like `gettid_inline` does this, too.